rotkehlchen/db/history_events.py

```diff
@@ -77,10 +77,11 @@
 def filter_ignore_asset_query(include_ignored_assets: bool = False) -> str:
     """Create and return the subquery to filter ignored assets. If `include_ignored_assets`
     is true then the filter is returned to include them."""
-    ignored_asset_subquery = "SELECT value FROM multisettings WHERE name='ignored_asset')"
+    subquery = "SELECT 1 FROM multisettings WHERE name = 'ignored_asset' AND value = asset"
```

Comment:
what's this change for?

Reply:
I measured it while doing the other changes and it was a bit faster but since I measured with other changes too I'll undo this until the follow up PR gets merged